README.md

```diff
@@ -45,11 +45,14 @@ FLAGS:
         --github-pre-lang    Use GitHub-style <pre lang> for code blocks
         --hardbreaks         Treat newlines as hard line breaks
     -h, --help               Prints help information
+        --no-config-file     Do not read config file
```

Comment:
What happens in the case of `--no-config-file --config-file ./config`? I assume `--no-config-file` takes precedence? Would it be just as well to have `--config-file false`?

Reply:
Yep, `--no-config-file` takes precedence. Hesitant to use `false` since it's not a boolean per se, but I do like not having the extra option, so will make `none` recognised!